money-monkey-web/components/SignOut.tsx

```
@@ -0,0 +1,20 @@
+import { useRouter } from "next/navigation";
+
+export default function SignOut() {
+  const supabase = createClient();
+  const router = useRouter();
+  return (
+    <form>
+      <button
+        onClick={() => {
+          supabase.auth.signOut().then(() => router.push("/login"));
+        }}
+      >
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)